core/src/main/scala/kafka/tools/ConsoleConsumer.scala

```diff
@@ -386,9 +386,15 @@ object ConsoleConsumer extends Logging {
       else if (fromBeginning) OffsetRequest.EarliestTime
       else OffsetRequest.LatestTime
 
-    if (!useOldConsumer)
+    if (!useOldConsumer) {
       CommandLineUtils.checkRequiredArgs(parser, options, bootstrapServerOpt)
 
+      if (options.has(newConsumerOpt)) {
+        Console.err.println("Using the --new-consumer option is deprecated and will be removed " +
+          "in a future major release. For using the new consumer just use only the --bootstrap-server option")
```

Comment:
Maybe:

"The --new-consumer option is deprecated and will be removed in a future major release. The new consumer is used by default if the --bootstrap-server option is provided."